irohad/ametsuchi/impl/postgres_wsv_query.cpp

```diff
@@ -284,16 +295,16 @@ namespace iroha {
           (sql_.prepare << "SELECT public_key, address FROM peer");
       std::vector<std::shared_ptr<shared_model::interface::Peer>> peers;
 
-      auto results = transform<
```

Comment:
Is 'transform' from postgres_wsv_common.hpp called from somewhere? Can the method be removed?